test/test_on_gpu/test_torch_model.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import numpy as np
+import torch
+import pytest
+
+from gluonts.dataset.pandas import PandasDataset
+from gluonts.evaluation import make_evaluation_predictions
+from gluonts.torch.model.estimator import PyTorchLightningEstimator
+from gluonts.torch.model.simple_feedforward import SimpleFeedForwardEstimator
+from gluonts.torch.model.deepar import DeepAREstimator
+from gluonts.evaluation import Evaluator
+
+
+@pytest.mark.parametrize(
+    "torch_model_cls",
+    [
+        SimpleFeedForwardEstimator,
+        DeepAREstimator
+    ]
+)
+def test_torch_model_on_gpu(
```

Comment:
You can condition the execution of this test on `torch.cuda.is_available()` using `skipif` from `pytest`: https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1